src/components/CompareResults/beta/RevisionRowExpandable.tsx

```diff
@@ -9,10 +9,17 @@ import Distribution from './Distribution';
 const strings = Strings.components.expandableRow;
 const { singleRun, note } = strings;
 
+function displayGraphDistribution(baseRuns: Array<number>, newRuns: Array<number>): boolean {
```

Comment:
because this returns a boolean, it's better to name it like a question, for example `shouldDisplayGraphDistribution`